pkg/controllers/user/helm/common/common.go

```diff
@@ -124,8 +135,18 @@ func InstallCharts(tempDirs *HelmPath, port string, obj *v3.App) error {
 	if err != nil {
 		return err
 	}
+	fmt.Print(tempDirs)
+	err = createKustomizeYaml(tempDirs, obj.Name)
```

Comment:
When obj.Spec.HelmVersion != "helm_v3", do we need to createKustomizeYaml? Should this be invoked only for helm_v3?